src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -1,17 +1,32 @@
 import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
 
-// Simplify relationship to a single key/value.
-function transformRelationship(relationship) {
-  let ret;
-  if (typeof relationship === 'string') {
-    ret = relationship; // Nothing to do here
-  } else if (relationship?.relationshipToVeteran) {
-    // If they chose "other" and typed a freeform answer, grab the text
-    ret = relationship.otherRelationshipToVeteran
-      ? relationship.otherRelationshipToVeteran
-      : relationship.relationshipToVeteran;
+// Rearranges date string from YYYY-MM-DD to MM-DD-YYYY
+function fmtDate(date) {
+  return date?.length && date.length === 10
+    ? `${date.slice(5)}-${date.slice(0, 4)}`
+    : date;
+}
```

Comment:
Convenience function to just dumbly swap characters in the date string. PDF expects it in MM-DD-YYYY, but the form outputs it in YYYY-MM-DD.